src/fibonacciBases.js

```diff
@@ -0,0 +1,48 @@
+function convertToFib() {
+  let number = document.getElementById('number').value
+  let result = []
+  let fibSeq = []
+
+//calculate fibSeq value
+  function fibValue(resultArr){
+    let value = 0
```

Comment:
Any time I see a code block that looks like this:
```js
// declare a variable
// iterate and mutate that variable
// return the variable
```
I tend to use the more concise, functional array utilities that javascript provides:
```js
fibSeq.reduce( (total, number, index) => total + number * resultArr[ index ], 0 )
```
Note that I renamed `value` - it led to confusion for me since it was hard to determine which thing we were looking at the value of in the functional approach (is it the current, aggregated value? the value in the array in a given iteration?)